### PR TITLE
Enabling disallow_untyped_calls in mypy.ini

### PR DESCRIPTION
**Pull request by jespino**

### CODE REVIEW

File: src/wallet/trade_manager.py [hunk 1]
```diff
@@ -532,7 +532,9 @@ async def respond_to_offer(self, file_path: Path) -> Tuple[bool, Optional[TradeR
         if chia_spend_bundle is not None:
             spend_bundle = SpendBundle.aggregate([spend_bundle, chia_spend_bundle])
             # debug_spend_bundle(spend_bundle)
-            if chia_discrepancy < 0:
+            if chia_discrepancy is None:
```
Comment (jespino) [inline src/wallet/trade_manager.py:535]: I don't know if this keeps the expected behavior.

File: src/wallet/trade_manager.py [hunk 2]
```diff
@@ -572,7 +574,9 @@ async def respond_to_offer(self, file_path: Path) -> Tuple[bool, Optional[TradeR
 
         for colour, amount in cc_discrepancies.items():
             wallet = wallets[colour]
-            if chia_discrepancy > 0:
+            if chia_discrepancy is None:
```
Comment (jespino) [inline src/wallet/trade_manager.py:577]: Same here

**Review by mariano54 — DISMISSED** [review]
awesome, thanks!

**Comment by hoffmang9** [comment]
This and its partner should be revived.

**Comment by jespino** [comment]
I'll try to rebase this during this weekend.

File: chia/wallet/trade_manager.py [hunk 3]
```diff
@@ -531,7 +531,9 @@ async def respond_to_offer(self, file_path: Path) -> Tuple[bool, Optional[TradeR
         now = uint64(int(time.time()))
         if chia_spend_bundle is not None:
             spend_bundle = SpendBundle.aggregate([spend_bundle, chia_spend_bundle])
-            if chia_discrepancy < 0:
+            if chia_discrepancy is None:
```
Comment (jespino) [inline chia/wallet/trade_manager.py:534]: I'm not 100% sure about this change, so please, somebody that understand better the intention of the code can check it?

File: chia/wallet/trade_manager.py [hunk 4]
```diff
@@ -571,7 +573,9 @@ async def respond_to_offer(self, file_path: Path) -> Tuple[bool, Optional[TradeR
 
         for colour, amount in cc_discrepancies.items():
             wallet = wallets[colour]
-            if chia_discrepancy > 0:
+            if chia_discrepancy is None:
```
Comment (jespino) [inline chia/wallet/trade_manager.py:576]: Same here.

**Comment by jespino** [comment]
@hoffmang9 revived! I merge main branch here, and added all the new types needed for work properly with the new config settings in the mypy.ini file.

**Pull request closed without merging**